plugins/woocommerce/includes/admin/meta-boxes/views/html-product-data-inventory.php

```diff
@@ -93,6 +93,17 @@
 			do_action( 'woocommerce_product_options_stock_fields' );
 
 			echo '</div>';
+		} else {
+
+			echo '<p class="form-field show_if_simple show_if_variable">';
+			echo '<label for="_manage_stock_disabled" aria-label="' . esc_attr__( 'Stock management disabled in store settings', 'woocommerce' ) . '">' . esc_attr__( 'Stock management', 'woocommerce' ) . '</label>';
+			echo '<output name="_manage_stock_disabled" id="_manage_stock_disabled" aria-live="off">' . sprintf(
+				/* translators: %s: url for store settings */
+				wp_kses( __( 'Disabled in <a href="%s" aria-label="stock management store settings">store settings</a>.', 'woocommerce' ), 'post' ),
+				esc_url( 'admin.php?page=wc-settings&tab=products&section=inventory' )
+			) . '</output>';
+			echo '</p>';
```

Comment:
What do you think about abstracting this a little bit? Maybe we can create a function `woocommerce_wp_label` [here](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/admin/wc-meta-box-functions.php).

Reply:
I was on the fence over whether to make a generic function for this at this point. But, I do agree it could have some value so I'll do this!

Reply:
Addressed in 7abc519